www/source/docs/reference/reference.html.md.erb

```diff
@@ -1260,13 +1260,45 @@ Packages are installed in the `/hab/pkgs/` directory, and then further organized
 ---
 ## <a name="sup-log-configuration" id="sup-log-configuration" data-magellan-target="sup-log-configuration">Supervisor Log Configuration Reference</a>
 
-With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are two main ways of configuring logging, each of which has their own strengths and weaknesses: using environment variables, and using a configuration file.
+With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are three ways of configuring logging, each of which has their own strengths and weaknesses: as part of the `hab` command, by using environment variables, and by using a configuration file.
 
-### Environment Variable Configuration
+### RUST_LOG with a command
+
+Use `RUST_LOG=debug RUST_BACKTRACE=1` to see an individual command's debug and backtrace.
```

Comment:
Might be worth a note that this ONLY works on mac/linux and not on Windows. However, I am actually in favor (not vehemently) of omitting this section altogether. This is really about setting command scoped environment variables which simply comes with the OS and is not Habitat specific.